Libs/Optimize/Optimize.cpp

```diff
@@ -228,6 +228,24 @@ int Optimize::GetDomainsPerShape()
   return this->m_domains_per_shape;
 }
 
+//---------------------------------------------------------------------------
+void Optimize::SetDomainType(std::string& domain_type)

```

Comment:
Should we use an enum or something rather than a std::string?

Reply:
This is the string it loads from the params file. At every point after this it uses the DomainType enum.

Reply:
Oh, you have a DomainType.  So it's a question of where the conversion from the parameter file's string takes place...

Reply:
Should we move it to OptimizeParameterFile.cpp?  I guess the mismatch of get/set is bothering me.

Reply:
Mismatch fixed